### PR TITLE
Support for Maven 3.3

### PR DESCRIPTION
**Pull request by iris-scrum-1**
Added support for Maven 3.3 by introducing redundant annotation based Mojo configuration for ResponderGenMojo + renaming some variables to match parameter name. Looks like regression in Maven 3.3 comparing to 3.0.5, but as the above changes help - probably no point investigating into Maven.

### CODE REVIEW

File: interaction-sdk-plugin/src/main/java/com/temenos/interaction/sdk/plugin/ResponderGenMojo.java [hunk 1]
```diff
@@ -41,26 +42,30 @@
     /**
      * @parameter property="edmxFile"
      */
-    private String edmxFileStr;
+    @Parameter(property = "edmxFile")
```
Comment (mtemenos) [inline interaction-sdk-plugin/src/main/java/com/temenos/interaction/sdk/plugin/ResponderGenMojo.java]: If I understand correctly the issue is that annotations in Javadoc are not getting processed correctly - I'm happy with that aspect of these changes. Please don't rename the edmxFileStr member as it reflects it's purpose and has caused unnecessary changes further down the file.

Comment (iris-scrum-1) [inline interaction-sdk-plugin/src/main/java/com/temenos/interaction/sdk/plugin/ResponderGenMojo.java]: OK, however the initial fix was the rename ☺ - was enough to make this work.
The Java 5 style annotations were added after to cover it “twice”

Will retest and publish again by Monday.

**Comment by mtemenos** [comment]
Thank you!

**Comment by iris-scrum-1** [comment]
The only way to workaround the issue known up to dayte is renaming the field in the Mojo to match the desired parameter name. https://issues.apache.org/jira/browse/MNG-5948 filled, and recorded in source comment for future tracking.

Redundant annotations (Javadoc and Java) were left in place, as they do no harm, and should increase stability if the javadoc comments get deprecated.
